cloudinit/distros/raspberry-pi-os.py

```diff
@@ -0,0 +1,107 @@
+# Copyright (C) 2024 Raspberry Pi Ltd. All rights reserved.
+#
+# Author: Paul Oberosler <paul.oberosler@raspberrypi.com>
+#
+# This file is part of cloud-init. See LICENSE file for license information.
+
+import os
+import shutil
+
+from cloudinit import subp
+from cloudinit.distros import debian
+
+
+class Distro(debian.Distro):
```

Comment:
At the very least some happy-path testing of the method overrides would be good here.